agent/lib/kontena/models/service_pod.rb

```diff
@@ -37,7 +37,8 @@ class ServicePod
                   :pid,
                   :hooks,
                   :secrets,
-                  :networks
+                  :networks,
+                  :volume_specs
```

Comment:
To distinguish between the volumes for the container (bind mounts, anon volumes, etc.) and the volumes which existence needs to be ensured when creating the container